docs/en/pact-reference.md

```diff
@@ -965,7 +965,7 @@ The execution request yaml for a public blockchain takes the following keys:
     gasLimit: integer gas limit
     gasPrice: decimal gas price
     ttl: integer time-to-live value
-    creationTime: integer tx execution time after offset
+    creationTime: optional integer tx execution time after offset
```

Comment:
I think this is incorrect. Are you thinking of `ApiReq`? This is the API, not `pact -a` etc.

Reply:
See https://github.com/kadena-io/pact/blob/70a6957b96d385190d0c3d75a31584373c9fbafd/src/Pact/Types/ChainMeta.hs#L113 -- it's required in datatype and JSON spec

Reply:
nm it's just L211 above